app/views/vatDetails/Details.scala.html

```diff
@@ -35,7 +35,7 @@
       vatOptOutSection: VatOptOutSection,
```

Comment:
Small thing but there's an import at the top for the covid partial (line 20) & it's also included in the main template (line 27) - these can be removed